bbc/simorgh
# Persian pashto nav updates to add Features and Radio indexes

Resolves JIRA n/a

Overall changes
======
- Added Evergreen features index to the Persian nav
- Added Radio programmes index to the Pashto nav

Code changes
======

- Edited Navigation section of src/app/lib/config/services/persian.ts to add Evergreen features index
- Edited Navigation section of src/app/lib/config/services/pashto.ts to add Radio programmes index
- Updates snapshots


Testing
======
1. Open Persian front page, last link on nav should point to /persian/topics/cxr3ex12k6et
2. Open Pashto front page, last link on nav should point to  /pashto/topics/c9xz1ekw79nt


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
